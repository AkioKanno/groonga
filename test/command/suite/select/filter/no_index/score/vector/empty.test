table_create Records TABLE_HASH_KEY ShortText
column_create Records tags COLUMN_VECTOR ShortText

load --table Records
[
{"_key": "empty", "tags": []}
]

select Records \
  --output_columns '_id, tags || "default"' \
  --command_version 2
