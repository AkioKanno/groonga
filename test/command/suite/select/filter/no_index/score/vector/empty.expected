table_create Records TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Records tags COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
load --table Records
[
{"_key": "empty", "tags": []}
]
[[0,0.0,0.0],1]
select Records   --output_columns '_id, tags || "default"'   --command_version 2
[[0,0.0,0.0],[[[1],[["_id","UInt32"],["tags","ShortText"]],[1,[]]]]]
